src/main/java/frc/robot/AutoCommands.java

```diff
@@ -22,22 +25,49 @@ public class AutoCommands {
     private final Swerve swerve;
     public final Map<String, SequentialCommandGroup> autos;
 
+    private final SendableChooser<SequentialCommandGroup> autoChooser;
+
     /**
      * Define all auto commands.
      */
     public AutoCommands(Swerve swerve) {
         this.swerve = swerve;
 
         autos = new HashMap<String, SequentialCommandGroup>();
+        autoChooser = new SendableChooser<>();
 
         autos.put("nothing", new SequentialCommandGroup(new InstantCommand(() -> {
```

Comment:
i already brought up that we should switch to using the `Commands` shorthands

Reply:
i will bring it up sat prog meeting, some ppl had thoughts on Tuesday about command factories so will  wait till sat to make changes to how we do commands

Reply:
the `Commands` class isn't about command factories really, they're just [shorthands to making code cleaner](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Commands.html)

Reply:
yea ik, i just dont want to change how we structure stuff without at least telling other ppl how the  new stuff works

Reply:
this pr already does exactly that, i dont really see your point

Reply:
this pr resturecurues code, however their is nothing new introduced here, I am restrucing code and if anybody on prog 1 pulled my new code they would be able to quickly understand what is happening, people are not as aware about command shorthands, if they pulled code with them they would be confused and would have to look up documentation and spend meeting time learning about how they work as such I don't want to introduce them until prog 1 people are familiar with them

Reply:
good point